src/core/errors/syscall_handler_errors.rs

```diff
@@ -19,7 +19,7 @@ pub enum SyscallHandlerError {
     FeltToUsizeFail,
     #[error("Couldn't convert Felt to u64")]
     FeltToU64Fail,
-    #[error("Couldn't compure hash")]
+    #[error("Couldn't compute hash")]
```

Comment:
This error is duplicated (see `ErrorComputingHash` a few lines below).

Reply:
this is why i add a single error in the fee module, when we add to many errors is hard to check whether what we are adding already exist or not.

Reply:
I always keep them alpha-sorted, so that they are easy to find later on.
Or by searching keywords project-wide (the tab between the project files and git in code/codium.